Sources/Subscription/AccountManager.swift

```diff
@@ -56,15 +56,15 @@ public class AccountManager: AccountManaging {
 
     public convenience init(subscriptionAppGroup: String?, accessTokenStorage: SubscriptionTokenStorage) {
         self.init(accessTokenStorage: accessTokenStorage,
-                  entitlementsCache: UserDefaultsCache<[Entitlement]>(subscriptionAppGroup: subscriptionAppGroup,
+                  entitlementsCache: UserDefaultsCache<[Entitlement]>(userDefaults: UserDefaults.init(suiteName: subscriptionAppGroup) ?? UserDefaults.standard,
```

Comment:
Not a fan of this. WDYT about moving back to defining 2 inits one for optional `subscriptionAppGroup` and other for tests to inject UserDefaults?

Reply:
I have removed `subscriptionAppGroup` because this class (UserDefaultsCache) is no longer part of subscription and adding that init would imply it's coupled.   This is now an abstract class, and makes more sense to inject a fully configured UserDefaults object (or let it default to Standard)